honeybee/model.py

```diff
@@ -373,6 +376,18 @@ def orphaned_shades(self):
         """Get a list of all Shade objects without parent Rooms in the model."""
         return tuple(self._orphaned_shades)
 
+    @property
+    def stories(self):
+        """Get a list of text for each unique story identifier in the Model.
+
+        Note that this will be an empty list if the model has to rooms.
+        """
+        _stories = set()
+        for room in self._rooms:
+            if room.story is not None:
+                _stories.add(room.story)
+        return list(_stories)
```

Comment:
I suggest to sort the list before returning it. Otherwise you can get them in a different order every time.